task16/index.js

```diff
@@ -261,18 +83,74 @@ const renderTabContainer = (value) => {
 const createArticleTitle = ({ article }) => {
   const articleTitleFragment = document.createDocumentFragment();
   for (let i = 0; i < article.length; i++) {
-    const articleTitleItem = createElementWithClassName("li", "article-title-item");
-    const articleWrapper = createElementWithClassName("article", "article-wrapper");
+    const articleTitleItem = createElementWithClassName(
+      "li",
+      "article-title-item"
+    );
+    const articleWrapper = createElementWithClassName(
+      "article",
+      "article-wrapper"
+    );
     const articleLink = createElementWithClassName("a", "article-link");
     const articleTitle = createElementWithClassName("h1", "article-title");
     articleTitle.textContent = article[i].title;
     articleLink.href = "#";
 
-    articleTitleFragment.appendChild(articleTitleItem).appendChild(articleWrapper).appendChild(articleLink).appendChild(articleTitle);
+    articleTitleFragment
+      .appendChild(articleTitleItem)
+      .appendChild(articleWrapper)
+      .appendChild(articleLink)
+      .appendChild(articleTitle);
+
+    renderNewArrivals(article[i], articleLink);
+    renderCommentCount(article[i], articleLink);
   }
   return articleTitleFragment;
 };
 
+const renderNewArrivals = (article, articleLink) => {
+  const articleDate = elapsedTime(article.date);
+  const newArrivalsLimiter = 3;
+  if (articleDate <= newArrivalsLimiter) {
+    articleLink.appendChild(createNewArrivals());
+  }
+};
+
+const renderCommentCount = (article, articleLink) => {
+  const commentLength = article.comment.length;
+  if (commentLength > 0) {
+    articleLink.appendChild(createCommentCount(article));
```

Comment:
IMO
```suggestion
    articleLink.appendChild(createCommentCount(commentLength));
```

Reply:
I felt it would be easier to read if I wrote it this way, but actually doing so resulted in an error.
The reason is that commentLength is set to determine the number of comments, and I felt that length alone was not enough information to execute createCommentCount.
Is this interpretation correct?

Reply:
すみません、意図を示すべきでした。

`createCommentCount()`内でも以下のように コメント数を格納した変数を定義していると思います。
```js
 const commentCount = commentValue.comment.length;
```
であれば`renderCommentCount()`内で定義している変数`commentLengthを`引数で渡せば、
`createCommentCount()`で定義する必要はなくなると思いコメントしました。
`createCommentCount()`内を適切に修正すれば、エラーがでないかと思います!

Reply:
さえさん解説いただきありがとうございます！！！
意図を間違えておりました😭
無事に修正できました！！感動してます😭大変勉強になりました！！！
d6dccfb